src/seed/seedUserDB.js

```diff
@@ -76,8 +76,8 @@ console.log(`build script `);
         password: 'psw123',
       };
       console.log(`User ${userData.email} has been `);
-      const user = new User(userData);
-      return user.save();
+      const user = User.create(userData);
+      return user;
```

Comment:
Could you send what you found on this? I have been reading and apparently `Model.create()` is the shorthand from 
```
 const user = new User(userData);
      return user.save();
```

Reply:
I did not read about  it in detail, we use it in the controller and i thought its shorter and creates all the defaults as well, that is why i use it. it works :)